model-archiver/model_archiver/model_packaging.py

```diff
@@ -38,7 +39,12 @@ def package_model(args, manifest):
         # Step 3 : Zip 'em all up
         ModelExportUtils.zip(export_file_path, model_name, model_path, files_to_exclude, manifest)
         logging.info("Successfully exported model %s to file %s", model_name, export_file_path)
-
+    except Exception as e:  # pylint: disable=broad-except
```

Comment:
except ModelArchiverError as e:
    logging.errror(e)
    sys.exit(1)

We don't need catch Exception, it will be printed with stacktrace.

Reply:
True, I used logging.exception() to print general exceptions with stacktrace. I will work on modifying this, but please do let me know if there is anything wrong with using logging.exception().